docs/getting_started/index.md

```diff
@@ -0,0 +1,16 @@
+# Getting Started
+
+Welcome to the **Getting Started** guide for Taipy. Taipy is composed of two components that enable anyone to create entire applications:
+
+- **Graphical User Interface builder** (Taipy GUI): allows any Python developer to create a complex and interactive GUI.
+
+- **Scenario Management** (Taipy Core): implements a modern backend for any data-driven application based on your business case.
+Three Getting Started are available. 
+
+Three **'Getting Started'** are available to introduce these two components. These 'Getting Started' are all **independent** and can be followed without any previous knowledge of Taipy.
+ 
+- [Getting Started with Taipy GUI](./getting-started-gui/index.md). It explains the basics of Taipy GUI. A multi-page NLP (Natural Language Processing) application is created along the way.
+
+- [Getting Started with Taipy Core](./getting-started-core/index.md). It shows how to configure and execute code with Taipy Core and all the useful features included.
+
+- [Getting Started with Core & GUI](./getting-started/index.md). This Getting Started has similarities with the previous aones. It is not necessary to have completed them before. It will use the power of both components to create a full Machine Learning application with just a few steps.
```

Comment:
Typo 'aones'
Did you run Grammarly on that?

Reply:
I have before I changed this exact line

Reply:
```suggestion
- [Getting Started with Core & GUI](./getting-started/index.md). This Getting Started has similarities with the previous ones. It is not necessary to have completed them before. It will use the power of both components to create a full Machine Learning application with just a few steps.
```